### PR TITLE
Fixes #6299: Improve SLES compatibility

### PR DESCRIPTION
**Pull request by Kegeruneku**
Ticket: http://www.rudder-project.org/redmine/issues/6299

### CODE REVIEW

File: scripts/add-repo.sh [hunk 1]
```diff
@@ -50,6 +50,7 @@ EOF
   then
     # Add SuSE repo
     rpm --import "http://keyserver.ubuntu.com/pks/lookup?op=get&fingerprint=on&search=0xADAB3BD36F07D355"
+    zypper removerepo Rudder || true
```
Comment (Kegeruneku) [inline scripts/add-repo.sh]: Not idempotent :)

File: scripts/setup-server.sh [hunk 2]
```diff
@@ -54,7 +61,7 @@ upgrade_server() {
   then
     ${PM_UPGRADE} rudder-server-root
   else
-    ${PM_UPGRADE} "rudder-*" "ncf"
+    ${PM_UPGRADE} "rudder-*" "ncf*"
```
Comment (Kegeruneku) [inline scripts/setup-server.sh]: According to the docs !

File: scripts/add-repo.sh [hunk 3]
```diff
@@ -50,6 +50,7 @@ EOF
...
     zypper addrepo -n "Normation RPM Repositories" "${URL_BASE}" Rudder || true
```
Comment (jooooooon) [inline scripts/add-repo.sh]: "Normation" RPM repos? I think you mean Rudder RPM Repositories

**Comment by peckpeck** [comment]
@Kegeruneku can you replace Normation in the Repo comment ?
Then you can self merge

**Comment by Kegeruneku** [comment]
PR has changed drastically and need a new review <3

**Pull request merged**